src/pages/Inventory.page.js

```diff
@@ -12,4 +12,49 @@ export class InventoryPage extends BaseSwagLabPage {
     async addItemToCartById(id) {
         await this.addItemToCartButton.nth(id).click();
     }
+
+    // Generate and return random number  generateRandomNumber(0, 6)  from 0 to 5 include.
+    static generateRandomNumber(min, max){
+        return  Math.floor( Math.random()*(max - min) + min)
+    }
+
+    //parse product data by index from Catalog and return an object
+    async parseItemDataByIndexCatalog(index){
+        const productItem =  this.page.locator('.inventory_item').nth(index); 
+        return {
+            name: await productItem.locator('.inventory_item_name').innerText(),
+            description: await productItem.locator('.inventory_item_desc').innerText(),
+            price: Number((await productItem.locator('.inventory_item_price').innerText()).replace('$', '')),
+            productIndex: index,
+        }
+    }
+
+    // generate two random number (can not be equal) and return result as array 
+    async generateTwoNotEqualRandomNumber(){
+        let numberOne = InventoryPage.generateRandomNumber(0,6)
+        let numberTwo = numberOne
+        
+        while (numberOne === numberTwo) {
+            numberTwo = InventoryPage.generateRandomNumber(0, 6);
+        }
+
+        return [numberOne,numberTwo];
```

Comment:
I doubt we should have such function or the other one (generateRandomNumber). The issue is in reusability of these functions. Its better to create function getRandomNumbersArray(max: number, length: number, min: number = 0). In this case function would be flexible and reusable.